BackofficeBundle/Resources/config/transformer.yml

```diff
@@ -1,22 +1,15 @@
 parameters:
-    open_orchestra_backoffice.transformer.embedkeywords_to_keywords.class: OpenOrchestra\Backoffice\Form\DataTransformer\EmbedKeywordsToKeywordsTransformer
     open_orchestra_backoffice.transformer.video_url_to_id.class: OpenOrchestra\Backoffice\Form\DataTransformer\VideoUrlToIdTransformer
     open_orchestra_backoffice.transformer.choices_option_to_array.class: OpenOrchestra\Backoffice\Form\DataTransformer\ChoicesOptionToArrayTransformer
     open_orchestra_backoffice.transformer.choice_array_to_string.class: OpenOrchestra\Backoffice\Form\DataTransformer\ChoiceArrayToStringTransformer
     open_orchestra_backoffice.transformer.choice_string_to_array.class: OpenOrchestra\Backoffice\Form\DataTransformer\ChoiceStringToArrayTransformer
     open_orchestra_backoffice.transformer.bbcode_to_html.class: OpenOrchestra\Backoffice\Form\DataTransformer\BBcodeToHtmlTransformer
     open_orchestra_backoffice.transformer.reference_to_embed.class: OpenOrchestra\Backoffice\Form\DataTransformer\ReferenceToEmbedTransformer
     open_orchestra_backoffice.transformer.block_to_array.class: OpenOrchestra\Backoffice\Form\DataTransformer\BlockToArrayTransformer
-
+    open_orchestra_backoffice.transformer.csv_to_reference_keyword.class: OpenOrchestra\Backoffice\Form\DataTransformer\CsvToReferenceKeywordTransformer
+    open_orchestra_backoffice.transformer.condition_to_reference_keyword.class: OpenOrchestra\Backoffice\Form\DataTransformer\ConditionToReferenceKeywordTransformer
+    
 services:
-    open_orchestra_backoffice.transformer.embedkeywords_to_keywords:
```

Comment:
@alavieille , open_orchestra_model.document.embed_keyword.class don't exist anymore. Should i add it too?